.github/workflows/client-targets.yml

```diff
@@ -34,6 +34,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Perl
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y perl
+
+      - name: Setup environment for Android NDK
+        run: |
+          echo "RANLIB=$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ranlib" >> $GITHUB_ENV
```

Comment:
Can you add a comment to remove this once we upgrade to a newer version of cargo-ndk? This was resolved awhile ago: https://github.com/bbqsrc/cargo-ndk/issues/90, and I confirm that it isn't needed with version 3.5.4

Reply:
resolved in [modify: client-targets.yaml](https://github.com/anza-xyz/agave/pull/3152/commits/cc04f9f7d4fd59e6a02519bc02085466829ddbfa)